.prettierrc.json

```diff
@@ -1,10 +1,10 @@
 {
-  "printWidth": 80,
+  "printWidth": 120,
   "tabWidth": 2,
   "useTabs": false,
   "semi": false,
   "singleQuote": true,
-  "trailingComma": "none",
-  "bracketSpacing": false,
-  "arrowParens": "avoid"
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "arrowParens": "always"
```

Comment:
This is the one thing I disagree with in these rules... 😂

Reply:
All 3 of these rules? I'm open to changing them. I like trailing comma because adding a line later is 1 less line of diff. I feel that bracket spacing just looks nice. Arrow parens I'm most on the fence about but I went with `always` because it means that when you add a 2nd+ param, you don't have to then add parens as well.

Reply:
specifically `arrowParens`. I understand the consistency angle, I just prefer the simplicity of no parens for single parameter methods. We can keep it as is :)